raiden/network/transport/udp/healthcheck.py

```diff
@@ -16,6 +16,10 @@
 from raiden.utils import pex
 from raiden.utils.typing import Address, Dict
 
+if TYPE_CHECKING:
```

Comment:
Some files have these guys at the bottom and some at the top, I moved them up because they are imports and to try to get a consistent way of doing this